src/Uno.UI/UI/Xaml/Internal/InputManager.Pointers.Managed.cs

```diff
@@ -387,6 +393,13 @@ private void OnPointerPressed(Windows.UI.Core.PointerEventArgs args)
 
 		private void OnPointerReleased(Windows.UI.Core.PointerEventArgs args)
 		{
+			// When multiple mouse buttons are pressed and then released, we only respond to the last OnPointerReleased
+			// (i.e when no more buttons are still pressed).
+			if (args.CurrentPoint.PointerDeviceType == PointerDeviceType.Mouse && args.CurrentPoint.IsInContact)
```

Comment:
I'm not sure that all pointer source properly sets the IsInContact, we need to validate on all platform.

Reply:
I checked the skia targets and it's set in all of them, so it should be fine.